sos/report/plugins/host.py

```diff
@@ -20,6 +20,8 @@ class Host(Plugin, RedHatPlugin, DebianPlugin):
 
     def setup(self):
 
+        self.add_forbidden_path('/etc/sos/default_mapping')
```

Comment:
Great catch!

Just.. isn't the path `/etc/sos/cleaner/default_mapping` instead?

Reply:
Yup, I forgot we moved that into the `cleaner` subdir. Updating these shortly.